rel-1.0.0/debian/core-deps/Dockerfile

```diff
@@ -3,6 +3,7 @@ FROM debian:jessie
 # Install .NET Core dependencies
 RUN apt-get update \
     && apt-get install -y --no-install-recommends \
+        ca-certificates \
```

Comment:
I agree that your proposal makes the most sense for this Dockerfile. I've updated it to match that.